hello-typescript/src/sort-array.ts

```diff
@@ -0,0 +1,19 @@
+interface DataItem {
+  createdDate: string;
+  modifiedDate: string;
+}
+
+export default function sortArray(array: DataItem[]): DataItem[] {
+const newArray = array;
```

Comment:
از آرایه‌ی اصلی باید کپی بگیرید
```suggestion
const newArray = [...array];
```